.openshiftio/application.yaml

```diff
@@ -117,14 +129,15 @@ objects:
         labels:
           app: thorntail-health-check
           group: io.openshift.boosters
+          version: ${RELEASE_VERSION}
       spec:
         containers:
         - env:
           - name: KUBERNETES_NAMESPACE
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
-          image: thorntail-health-check:latest
+          image: ""
```

Comment:
Looks weird on the first sight...

Reply:
If we do not make it this way, 2 deployments were getting triggered.